tasks/scripts/format-slack-message-norsk

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e -x
+
+message_file="$PWD/message/message"
+echo $(jq -r '.jobs[] | select(.status == "failed") | .url' $PWD/src/$NORSK_PIPELINE/status.json) > $message_file
```

Comment:
unless the `echo` is intentional, I think we can get rid of it since in case of multiple failures, this will remove the newlines between each line.
```suggestion
jq -r '.jobs[] | select(.status == "failed") | .url' $PWD/src/$NORSK_PIPELINE/status.json > $message_file
```

Reply:
The echo was intentional, and the idea behind it was to list the urls to the jobs on the norsk pipeline that failed. The default slack message would lead us to our norsk-mirror job (which only contains a task that scans the status.json file for failed jobs). I wanted to make it easy to jump to the pipeline/job that actually failed by clicking on the slack message. But I guess you have a good point... if all jobs failed, we would have a slack message with probably more than 10 urls. 
Instead of removing the echo, I thought that I could include then only the url to the nork pipeline, not all those jobs that failed. What do you think? If you agree I'll submit changes to the PR later today or tomorrow.

Reply:
aaand it took me that long to respond 😁.
my idea was that the urls wouldn't have any separations between them so if we have multiple links it will be a bit unreadable.

Reply:
but yeah, maybe using another link could solve it as well.

Reply:
Good catch! Thanks!